pkg/unittest/test_suite.go

```diff
@@ -328,35 +332,44 @@ func (s *TestSuite) polishChartSettings(test *TestJob) {
 	log.WithField(common.LOG_TEST_SUITE, "polish-chart-settings").Debug("test.chart '", test.Chart)
 }
 
+type SuiteResult struct {
+	Pass       bool
+	FailFast   bool
+	JobResults []*results.TestJobResult
+}
+
 func (s *TestSuite) runV3TestJobs(
 	chartPath string,
 	cache *snapshot.Cache,
-	failfast bool,
+	failFast bool,
 	renderPath string,
-) (bool, []*results.TestJobResult) {
-	suitePass := false
+) *SuiteResult {
+	result := SuiteResult{Pass: true, FailFast: false}
 	jobResults := make([]*results.TestJobResult, len(s.Tests))
 
-	for idx, testJob := range s.Tests {
-		// (Re)load the chart used by this suite (with logging temporarily disabled)
-		log.SetOutput(io.Discard)
-		chart, _ := v3loader.Load(chartPath)
-		log.SetOutput(os.Stdout)
+	// (Re)load the chart used by this suite (with logging temporarily disabled)
+	log.SetOutput(io.Discard)
+	// TODO: is this required? As we are loading the chart in the test runner RunV3
```

Comment:
Reloading the chart is required to ensure the default values are reset back. Otherwise we risk values overridden by previous tests are reused.

Reply:
Gotcha. Moved back. Strange, that no tests are failing. Worth to consider in follow-up most likely to improve this line. As If there enormous number of tests, it could be reloaded quite a few times.